hbase_sql/src/main/java/com/splicemachine/olap/OlapServerSubmitter.java

```diff
@@ -130,6 +130,10 @@ public void run() {
             } else {
                 this.appStagingBaseDir = FileSystem.get(conf).getHomeDirectory();
             }
+            String yarnQueue = System.getProperty("splice.olapServer.yarn.queue");
```

Comment:
Is "splice.olapServer.yarn.queue" initialized anywhere (put in the property)? Or we are recommending to put olapServer to the default splice.spark.yarn.queue?